### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by JonnyWong16**
Knocking out a bunch of the easier feature requests.
#369, #184, #329, #293, #285, #291

### CODE REVIEW

File: plexpy/datafactory.py [hunk 1]
```diff
@@ -589,6 +589,38 @@ def get_home_stats(self, time_range='30', stats_type=0, stats_count='5', stats_c
                 home_stats.append({'stat_id': stat,
                                    'rows': last_watched})
 
+            elif stat == 'most_concurrent':
+                try:
+                    query = 'SELECT started, stopped ' \
+                            'FROM session_history '
```
Comment (drzoidberg33) [inline plexpy/datafactory.py]: You need to include the time range here.

Comment (JonnyWong16) [inline plexpy/datafactory.py]: I was going for all time, but I guess to keep it consistent with the rest of the stats....

Comment (drzoidberg33) [inline plexpy/datafactory.py]: Yes, please keep it consistent with the rest.

File: plexpy/__init__.py [hunk 2]
```diff
@@ -295,7 +295,7 @@ def initialize_scheduler():
 
             if CONFIG.MONITOR_REMOTE_ACCESS:
                 schedule_job(activity_pinger.check_server_response, 'Check for server response',
-                             hours=0, minutes=0, seconds=seconds)
+                             hours=0, minutes=0, seconds=2)
```
Comment (drzoidberg33) [inline plexpy/__init__.py]: What's going on here?

File: data/interfaces/default/js/tables/history_table.js [hunk 3]
```diff
@@ -265,6 +265,8 @@ history_table_options = {
                 createChildTable(this, rowData)
             }
         });
+
+        $("#history_table_info").append(' with a duration of ' + settings.json.filter_duration + ' (filtered from ' + settings.json.total_duration + ' total)');
```
Comment (drzoidberg33) [inline data/interfaces/default/js/tables/history_table.js]: If we're going to have this here please find a way to make it responsive. This is a huge line of text.

File: plexpy/__init__.py (hunk 2 above)
Comment (JonnyWong16) [inline plexpy/__init__.py]: Haha whoops. I changed it to test the notifications.

**Comment by JonnyWong16** [comment]
Merging. Will run dev branch for a few days.

**Pull request merged**